### PR TITLE
chore: Improve operator CSV

### PR DESCRIPTION
**Pull request by agranado2k**
Include a few more labels, annotations and spec for operator only YAML file CSV.

- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

Includes new labels and annotation as part of the operator certification process for IBM

### Notes for the reviewer

The only YAML file changed was the operator template.

### More information

- [RUN-999](https://snyksec.atlassian.net/browse/RUN-999)

### CODE REVIEW

File: snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1,14 +1,30 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  labels:
+    app.kubernetes.io/instance: snyk-monitor
```
Comment (shaimendel) [inline snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml:5]: the constant "0.0.0" appears in multiple places here, is it supposed to stay "0.0.0" when we build the operator?

File: snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -1,14 +1,30 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  labels:
```
Comment (shaimendel) [inline snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml:4]: should all of these changed be reflected in the helm charts / yamls as well?

File: snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (agranado2k) [reply snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml:5]: No, we replace it for the current `kubernetes-monitor` version. You can take a look [here](https://github.com/snyk/kubernetes-monitor/blob/6dc63bc45bc8e3cbc58c3b67b77652e1cc2357d0/scripts/operator/package-operator.sh#L33).

File: snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml (hunk 2 above)
Comment (agranado2k) [reply snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml:4]: In my point of view it's just relevant for the operator certification process for IBM. It's not required in the charts / yamls. However we can include it there as complementary information, I can't see any problem. For my ambiguous answer I don't have preference. :)

**Pull request closed without merging**